### PR TITLE
(form) Update follow up form, adding other field to capture other ARV Regimen

### PR DESCRIPTION
**Pull request by Michaelndula**

### CODE REVIEW

File: distro/configuration/concepts/hiv_and_art_follow_up_concept_set.csv [hunk 1]
```diff
@@ -1,43 +1,67 @@
...
+5a = AZT/3TC+LPV/r;
+5b = AZT/3TC + RAL;
+5c = ABC/3TC (120/60) + RAL;
+5d = AZT/3TC + ATV/r;
+5e = ABC/3TC + ATV/r;
+5f = TDF/ 3TC + ATV/r;
+5g = AZT/3TC + DTG50;
+5h = ABC/3TC + DTG50;
+5i = ABC/3TC + LPV/r;
+5j: AZT/3TC (120/60)+DTG10;
```
Comment (moshonk) [inline distro/configuration/concepts/hiv_and_art_follow_up_concept_set.csv:49]: Make this format consistent

File: distro/configuration/concepts/hiv_and_art_follow_up_concept_set.csv [hunk 2]
```diff
@@ -1,43 +1,67 @@
...
+2g = AZT/3TC + LPV/r;
+2h = AZT/3TC + ATV/r;
+2i = ABC/3TC + LPV/r;
+2J  = ABC/3TC + ATV/r;
+2k = TDF/3TC/DTG;
+4a = AZT/3TC/NVP;
+4b = AZT/3TC + EFV;
+4c = ABC/3TC (120/60) + LPV/r;
+4d = ABC/3TC (120/60) + DTG50;
+4e: ABC/3TC (120/60)+DTG10;
```
Comment (moshonk) [inline distro/configuration/concepts/hiv_and_art_follow_up_concept_set.csv:32]: Make this format consistent with the rest

File: distro/configuration/concepts/hiv_and_art_follow_up_concept_set.csv (hunk 1 above)
Comment (Michaelndula) [reply distro/configuration/concepts/hiv_and_art_follow_up_concept_set.csv]: @moshonk this is resolved

File: distro/configuration/concepts/hiv_and_art_follow_up_concept_set.csv (hunk 2 above)
Comment (Michaelndula) [reply distro/configuration/concepts/hiv_and_art_follow_up_concept_set.csv]: @moshonk this is resolved

**Pull request merged**